internal/worker/queue.go

```diff
@@ -102,10 +105,13 @@ func (q *queue) Start(ctx context.Context) (<-chan error, error) {
 	return ech, nil
 }
 
+// isRunning returns true when queue is already running or false when queue is not running.
 func (q *queue) isRunning() bool {
 	return q.running.Load().(bool)
 }
 
+// Push sends JobFunc to q.inCh when JobFunc is not nil and queue is running.
```

Comment:
Please describe the function with understandable statement. 
Readers do not understand this statement if they don't read and understand the source code.